app/config.py

```diff
@@ -0,0 +1,80 @@
+import json
+import os
+
+from kombu import Exchange, Queue
+
+
+class QueueNames:
+    LETTERS = 'letter-tasks'
+    SANITISE_LETTERS = 'sanitise-letter-tasks'
+
+
+class TaskNames:
+    PROCESS_SANITISED_LETTER = 'process-sanitised-letter'
+    UPDATE_BILLABLE_UNITS_FOR_LETTER = 'update-billable-units-for-letter'
+    UPDATE_VALIDATION_FAILED_FOR_TEMPLATED_LETTER = 'update-validation-failed-for-templated-letter'
+
+
+class Config:
+    NOTIFY_ENVIRONMENT = os.environ['NOTIFY_ENVIRONMENT']
+
+    AWS_REGION = 'eu-west-1'
+    TEMPLATE_PREVIEW_INTERNAL_SECRETS = json.loads(
+        os.environ.get('TEMPLATE_PREVIEW_INTERNAL_SECRETS', '[]')
+    )
+    NOTIFY_APP_NAME = 'template-preview'
+    DANGEROUS_SALT = os.environ.get('DANGEROUS_SALT')
+    SECRET_KEY = os.environ.get('SECRET_KEY')
+
+    CELERY = {
+        'broker_url': 'sqs://',
+        'broker_transport_options': {
+            'region': AWS_REGION,
+            'visibility_timeout': 310,
+            'wait_time_seconds': 20,  # enable long polling, with a wait time of 20 seconds
+            'queue_name_prefix': ({
+                'test': 'test',
+                'development': os.environ.get('NOTIFICATION_QUEUE_PREFIX', 'development'),
```

Comment:
i don't think we should default to `'development'` as that'll leave people potentially using that variable and overwriting each other. Ideally we should error but that might not be easy to write since this is all executed at module level on every env 🤔. In other apps we just let it default to `None`.

Reply:
Sure, changed in https://github.com/alphagov/notifications-template-preview/pull/651/commits/d754c2d3c7300a09efa5d41386f6c82748a6a6dd.

Reply:
ahhhhh the live production mixup strikes again!